content-src/lib/selectLayoutRender.js

```diff
@@ -94,4 +106,24 @@ export const selectLayoutRender = (state, prefs, rickRollCache) => {
       return {...component, data};
     }),
   })).filter(row => row.components.length);
+
+  let spocsFill = [];
```

Comment:
What do we want to sent if spocs are not enaled? Right now I think you return `[]` if spocs are not enabled, does that make sense?

Reply:
In `DiscoveryStreamBase.js`, it'll only send this ping if `spocsFill` is not empty.